exchanges/subscription/fixtures_test.go

```diff
@@ -65,7 +62,14 @@ func (m *mockEx) GetSubscriptionTemplate(s *Subscription) (*template.Template, e
 				ap[asset.Spot] = ap[asset.Spot][0:1]
 				return ""
 			},
-			"batch": common.Batch[currency.Pairs],
+			"batch": func(pairs currency.Pairs, size int) []string {
+				s := []string{}
+				for _, p := range common.Batch(pairs, size) {
+					p = p.Format(currency.PairFormat{Uppercase: true})
```

Comment:
Question: Wouldn't this purge incoming formats? Like maybe they require delimiter in batches as an edge case?

Reply:
nevermind just noticed this as being in a test file, silly billy.